src/components/Comments.jsx

```diff
@@ -32,6 +34,6 @@ export const Comments = () => {
                 <p className="comment-body">{comment.body}</p>
             </li>
```

Comment:
Lines 29 - 35 definitely could be extracted into another component to stop this component from bloating up